scout/server/blueprints/clinvar/static/clinvar_submissions.js

```diff
@@ -0,0 +1,35 @@
+async function getJson(url) {
```

Comment:
You probably wrote this code mostly in  javascript because you are more familiar with this language, but I think the whole download thing should be handled by python instead, being scout flask project.. Of course we use javascript in some parts, but it's minor things in general, like form checks etc. It's going to be a good exercise I think! ;)